src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -549,6 +562,15 @@ function MoneyRequestView({
                         />
                     </OfflineWithFeedback>
                 )}
+                {shouldShowViewTripDetails && tripID && (
```

Comment:
if we need tripID to Show the Trip Details, should we include `&& tripID` in shouldShowViewTripDetails declaration

Reply:
yeah that makes sense